recipes/chartify/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "chartify" %}
+{% set version = "2.3.4" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 3f991749eb85c90bb527eb08f242799325ad0f891f3744c9b2ed653fa54ece50
+
+build:
+  number: 0
+  skip: True  # [py2k]
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+
+requirements:
+  host:
+    - bokeh >=0.12.16,<2.0.0
```

Comment:
Is this a build time dep?

Reply:
The build failed when I tried to build it as a noarch package.
https://circleci.com/gh/conda-forge/staged-recipes/31783?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

Reply:
Probably not. I'll remove it.